interop/src/build/web/cryptobox-esm/.gitignore

```diff
@@ -0,0 +1 @@
+bun.lock
```

Comment:
I'm wondering, why don't we just drop the whole cryptobox client from interop and related things?
We don't really support it anymore, we just have the migration, and even that is going to go away soon.